client/coral-embed-stream/src/Embed.js

```diff
@@ -144,7 +158,10 @@ class Embed extends Component {
       <div style={expandForLogin}>
         <div className="commentStream">
           <TabBar onChange={this.changeTab} activeTab={activeTab}>
-            <Tab><Count count={asset.totalCommentCount}/></Tab>
+            <Tab><Count
+              count={asset.totalCommentCount}
+              handleClick={this.handleClick}/>
```

Comment:
No it's perfect there. The `isAdmin` was a merge mistake. It should never have been there. 
Just remove the  `isAdmin` and put the `his.props.viewAllComments();` over `refetch` and everything should work.